Makefile

```diff
@@ -19,7 +19,7 @@ Q = $(if $(filter 1,$V),,@)
 M = $(shell printf "\033[34;1m▶\033[0m")
 
 .PHONY: all
-all: fmt lint vendor | $(BASE) ; $(info $(M) building executable…) @ ## Build program binary
+all: fmt vendor | $(BASE) ; $(info $(M) building executable…) @ ## Build program binary
```

Comment:
Are you really sure you want to remove all `lint`ing? Like really really sure? Probably would be a better idea to fix the code so that it goes through golint.

Reply:
Well the issue was with vendor includes, ill look into disabling for includes